### PR TITLE
Add labels from message to github createIssue

### PR DESCRIPTION
**Pull request by agius**
Solves #92 - also updates our Github dependency from the deprecated [github npm package](https://www.npmjs.com/package/github) to [octokit/rest.js](https://github.com/octokit/rest.js)

### CODE REVIEW

File: incoming/function.js [hunk 1]
```diff
@@ -337,6 +337,15 @@ incoming.callGitHub = function(user, message, requestId, gitHubOwner, gitHubRepo
     title: message.body.github.title
   };
 
+  // labels can be passed in as an array of strings, and said labels will be applied to
+  // the new issue. any labels that don't already exist in that repo will be created
+  // https://developer.github.com/v3/issues/#create-an-issue
```
Comment (k-mahoney) [inline incoming/function.js]: Awesome, glad we can create them on demand 💯

File: package.json [hunk 2]
```diff
@@ -24,12 +24,12 @@
   "dependencies": {
     "@mapbox/lambda-cfn": "^2.0.0",
     "@mapbox/pagerduty": "^1.0.0",
+    "@octokit/rest": "^15.8.0",
```
Comment (k-mahoney) [inline package.json]: Thanks for grabbing this upgrade

**Comment by agius** [comment]
> GitHub simply needs to see them in the message body object? No changes necessary for lib/github.js?

yup! I'd gotten about ¾ the way through coding all that (`getIssues`, `findIssueByName`, etc) and then just tried it in my fav REST client, and it works the way we'd want it to here.

**Comment by k-mahoney** [comment]
Awesome - want to give it a test run on staging? Happy to help or answer any questions about throwing SNS messages around with the AWS CLI.

**Pull request merged**